src/components/UserActions/UserActions.ts

```diff
@@ -143,7 +143,9 @@ export class UserActions extends Component {
             (get(this.root, UserProfileModel) as UserProfileModel)
                 .getActions(this.options.userId)
                 .then(actions => (actions.length > 0 ? this.render() : this.renderNoActions()))
-                .catch(() => this.renderNoActions());
+                .catch(e => {
+                    e.statusCode === 404 ? this.renderEnablePrompt() : this.renderNoActions();
```

Comment:
mhhh.... it's between curly braces and the then l145 is between 'simple' braces '(', is that normal?

Reply:
Nice catch! The linter allows this to be on one line 👍